core/modules/layout/css/layout.admin.css

```diff
@@ -337,6 +337,22 @@
   }
 }
 
+/* Make sure that the machine name wraps under the "Admin label" field. */
+.layout-block-configure-form .form-item-label span.field-suffix {
```

Comment:
This CSS is now specific enough, it will target all suffixes for all fields on this form. If all we're after is the machine name, we should target that single item specifically.

Reply:
I've removed this CSS, and left it to be handled in https://github.com/backdrop/backdrop-issues/issues/1349